packages/picasso-codemod/src/v18.0.0/picasso-lab/picasso-lab.ts

```diff
@@ -0,0 +1,16 @@
+import { Transform } from 'jscodeshift'
+
+const transform: Transform = (file, api) => {
+  const j = api.jscodeshift
+
+  return j(file.source)
+    .find(j.StringLiteral)
```

Comment:
```suggestion
    .find(j.StringLiteral, ({ value }) => value.includes('picasso-lab'))
```
I'd use filtering here so that `replace` method is not used on every string literal

Reply:
I had `.filter` before, idk why I removed it. Good catch :+1: